Sources/SwifterSwift/Foundation/NSRegularExpressionExtensions.swift

```diff
@@ -81,10 +82,11 @@ public extension NSRegularExpression {
     /// - Returns: The number of matches of the regular expression.
     func numberOfMatches(in string: String,
                          options: MatchingOptions = [],
-                         range: Range<String.Index>) -> Int {
-        return numberOfMatches(in: string,
-                               options: options,
-                               range: NSRange(range, in: string))
+                         range: Range<String.Index>) -> Int
+    {
+        numberOfMatches(in: string,
+                        options: options,
+                        range: NSRange(range, in: string))
```

Comment:
This is what I meant before about redundant returns for functions. I find it very strange.